src/app/features/genes/components/gene-soe-charts/gene-soe-charts.component.scss

```diff
@@ -52,12 +52,12 @@
     margin: 0;
   }
 
-  .score-chart {
-    min-width: 320px;
-  }
-
-  .chart-body {
-    padding-top: 15px;
-    padding-bottom: 15px;
-  }
+  // .score-chart {
+  //   min-width: 320px;
+  // }
+
+  // .chart-body {
+  //   padding-top: 15px;
+  //   padding-bottom: 15px;
+  // }
```

Comment:
Delete if this won't be used in the near-future 🙂

Reply:
Might as well delete this since I think this whole chart will be rewritten.